package.json

```diff
@@ -109,6 +109,7 @@
     "d3": "^5.16.0",
     "detect-browser": "^3.0.1",
     "domready": "^0.3.0",
+    "fbjs": "^0.8.17",
```

Comment:
What's this be for?

Reply:
Oh yeah I think we discussed; this pinned fbjs was able to be removed on CGAP after [updating html-react-parser dependencies](https://github.com/dbmi-bgm/cgap-portal/pull/475). Not sure if same still needs to be done on 4DN-side... probably updated version is already on here since most of changes we in shared-portal-components.

The things that might need to do on 4DN-side is [update html-react-parser version](https://github.com/dbmi-bgm/cgap-portal/pull/475/commits/91a2d6ef2f8eadb8f49f0677ac709e63e7313229) (4DN is still on [old version in its package.json](https://github.com/4dn-dcic/fourfront/blob/master/package.json#L116)) and ~~[these minor updates to webpack.config.js](https://github.com/dbmi-bgm/cgap-portal/pull/475/commits/dd2e017e7d1c99285e26c9ffdfad070c82b7d1e0#diff-1fb26bc12ac780c7ad7325730ed09fc4c2c3d757c276c3dacc44bfe20faf166f)~~ (nvm -- these look like are in master already). (@utku-ozturk)